setup.py

```diff
@@ -53,7 +53,6 @@
     'snowballstemmer>=1.1',
     'babel>=1.3,!=2.0',
     'alabaster>=0.7,<0.8',
-    'sphinx_rtd_theme>=0.1,<2.0',
```

Comment:
It's not ok to remove this theme without deprecation period, since some folks may rely on it.